src/App.jsx

```diff
@@ -39,7 +39,11 @@ export const App = () => {
               <Route path="/sounds" element={<SoundsRoute />} />
               <Route path="/draft" element={<DraftRoute />} />
               <Route path="/articles" element={<ArticlesRoute />} />
-              <Route exact path="/articles/create" element={<ArticleRoute />} />
+              <Route
+                exact
+                path="/articles/create"
+                element={<ArticleRoute type={"new"} />}
```

Comment:
I incorporated the **type** prop, and am using it so that the delete button is always there on the 'edited' instance of the Article component, rather than being invisible for a moment. I figured it was slightly nicer this way. But if preferred the other way, I can easily change that.